### PR TITLE
add community accomodation services demo environment

### PR DESCRIPTION
**Pull request by Rosie-Brigham**
Adds a new demo environment for community accomodation services - the purpose of this is to create a stable environment in which to develop a really solid set of testing data that can be used for local testing where large amounts of (non prod) data are required.

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/resources/rds.tf [hunk 1]
```diff
@@ -0,0 +1,81 @@
+module "rds" {
+  source                       = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=6.0.1"
```
Comment (timckt) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/resources/rds.tf:2]: please update with 
```
source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=7.0.0"
```

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-approved-premises-test-cert
+  namespace: hmpps-community-accommodation-test
```
Comment (timckt) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml:6]: need to replace with `hmpps-community-accommodation-demo`

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+    - approved-premises-test.hmpps.service.justice.gov.uk
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-temporary-accommodation-test-cert
+  namespace: hmpps-community-accommodation-test
```
Comment (timckt) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml:19]: need to replace with `hmpps-community-accommodation-demo`

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    kind: ClusterIssuer
+  dnsNames:
+    - temporary-accommodation-test.hmpps.service.justice.gov.uk
+    - transitional-accommodation-test.hmpps.service.justice.gov.uk
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-approved-premises-api-demo-cert
+  namespace: hmpps-community-accommodation-test
```
Comment (timckt) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml:33]: need to replace with `hmpps-community-accommodation-demo`

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+    - approved-premises-api-test.hmpps.service.justice.gov.uk
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-community-accommodation-wiremock-test-cert
+  namespace: hmpps-community-accommodation-test
```
Comment (timckt) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml:46]: need to replace with `hmpps-community-accommodation-demo`

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+    - community-accommodation-wiremock.hmpps.service.justice.gov.uk
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-community-accommodation-tier-2-test-cert
+  namespace: hmpps-community-accommodation-test
```
Comment (timckt) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/06-certificates.yaml:59]: need to replace with `hmpps-community-accommodation-demo`

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/resources/serviceaccount-circleci.tf [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  namespace = var.namespace
+  kubernetes_cluster = var.kubernetes_cluster
+
+  serviceaccount_token_rotated_date = "01-01-2000"
+
+  serviceaccount_name = "circleci-migrated"
+
+  # Uncomment and provide repository names to create github actions secrets
+  # containing the ca.crt and token for use in github actions CI/CD pipelines
+  # github_repositories = ["my-repo"]
```
Comment (williamfalconeruk) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/resources/serviceaccount-circleci.tf:13]: does this need to be uncommented?

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/resources/irsa.tf [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+  }
+  sqs_queues_cas2 = {
+    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-demo",
+  }
+  sqs_queues_ta = {
+    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-demo",
+  }
+  sns_topics = {
+    "cloud-platform-Digital-Prison-Services-e29fb030a51b3576dd645aa5e460e573" = "hmpps-domain-events-demo"
+  }
```
Comment (williamfalconeruk) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/resources/irsa.tf:13]: see this line as example on our test environment - we only have event and audit queues for dev, preprod and prod as far as i can see for all services.
 - i think these need to point to dev audit and domain event queues ot get this functioning.
```suggestion
  sqs_queues_ap = {
    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-dev",
  }
  sqs_queues_cas2 = {
    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-dev",
  }
  sqs_queues_ta = {
    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-dev",
  }
  sns_topics = {
    "cloud-platform-Digital-Prison-Services-e29fb030a51b3576dd645aa5e460e573" = "hmpps-domain-events-dev"
  }
```

**Pull request merged**